web/src/app/browser/src/utils/rotationProcessor.ts

```diff
@@ -58,6 +58,15 @@ export class RotationProcessor {
       window.clearInterval(this.updateTimer);
       this.rotState = null;
     }
+
+    const target = this.keyman.contextManager.activeTarget;
+    if(target) {
+      // This seems to help with scrolling accuracy in iOS Safari;
+      // the scroll tends to consistently go too far without it.
+      window.setTimeout(() => {
+        this.keyman.ensureElementVisibility(target.getElement());
+      }, 0);
```

Comment:
Possibly relevant:  I can't help but notice what _while the device is rotating_ with my personal iPhone, the font-size for text components of the page increases notably when rotating from "portrait" to "landscape".  Certain parts then "snap back" to normal when the rotation is done.
- The top, black "title" text grows but snaps back
- The red text before each input element and text area actually retains its size increase.

If these adjustments aren't fully triggering until the rotation handler is complete, that might explain the need for the timeout.

Reply:
Which page are you referring to? Can you capture a video (even shakycam if necessary)?